python_modules/dagster/dagster/_utils/container.py

```diff
@@ -0,0 +1,108 @@
+import logging
+import os
+from typing import Optional, TypedDict
+
+import pendulum
+
+
+def cpu_usage_path():
+    return os.getenv("DAGSTER_CPU_USAGE_PATH", "/sys/fs/cgroup/cpuacct/cpuacct.usage")
+
+
+def cpu_info_path():
+    return os.getenv("DAGSTER_CPU_INFO_PATH", "/proc/cpuinfo")
```

Comment:
Could just use `psutil.Process().cpu_affinity()`: https://psutil.readthedocs.io/en/latest/#psutil.cpu_count

Reply:
gonna keep it with the file, just keeps the mocks consistent.